### DISCUSSION TITLE
Check out WKWebView's App-Bound Domains?

### DISCUSSION THREAD

**Discussion by dminkovsky** [ionic-team/capacitor#3548] [General]
I was trying to figure out whether a WKWebView can use a Service Worker. Sounds like they cannot. But maybe soon?

https://webkit.org/blog/10882/app-bound-domains/ via https://twitter.com/othermaciej/status/1295434201865412608

**Comment by FluorescentHallucinogen** [comment]
Any news?

It seems it's easy as adding

```
<key>WKAppBoundDomains</key>
<array>
  <string>example.com</string>
  <string>a.example.com</string>
  <string>b.example.com</string>
</array>
```

(up to 10 domains) into `Info.plist` file.

**Comment by FluorescentHallucinogen** [comment]
@jcesarmobile @dwieeb @mhartington @mlynch @ikeith PTAL. :wink:

**Reply by ikeith** [reply]
https://github.com/ionic-team/capacitor/issues/4122

**Comment by maxcodefaster** [comment]
Has anybody gotten service workers to work in WKWebView?

I set limitsNavigationsToAppBoundDomains = true in the capacitor config and added 
<key>WKAppBoundDomains</key>
<array>
  <string>loaclhost</string>
</array>

But navigator.serviceWorker.register fails
